src/pug/index.pug

```diff
@@ -64,7 +64,7 @@ html(lang='en')
                     .subheading.mb-5
                         | Computer Architecture · Design Automation · Machine Learning ·
                         |
-                        a(href='mailto:cepiross@korea.ac.kr') cepiross at korea.ac.kr
```

Comment:
https://ic.korea.ac.kr/ic/inquiry/notice.do?mode=view&articleNo=301315
According to the notice, they will terminate KU email accounts for alumni by September 26, 2022. Any alumni membership and/or its support are TBD :rofl: